[2025-04-18T04:02:15.937+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T04:02:15.954+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.clean_data scheduled__2025-04-18T03:00:00+00:00 [queued]>
[2025-04-18T04:02:15.964+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.clean_data scheduled__2025-04-18T03:00:00+00:00 [queued]>
[2025-04-18T04:02:15.965+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-18T04:02:15.982+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): clean_data> on 2025-04-18 03:00:00+00:00
[2025-04-18T04:02:15.993+0000] {standard_task_runner.py:72} INFO - Started process 981 to run task
[2025-04-18T04:02:16.000+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'clean_data', 'scheduled__2025-04-18T03:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpr1m88yx6']
[2025-04-18T04:02:16.004+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask clean_data
[2025-04-18T04:02:16.030+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-18T04:02:16.064+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.clean_data scheduled__2025-04-18T03:00:00+00:00 [running]> on host e8dafad5cd22
[2025-04-18T04:02:16.067+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-18T04:02:16.157+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='clean_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-18T03:00:00+00:00'
[2025-04-18T04:02:16.159+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T04:02:16.179+0000] {data_pipeline.py:51} INFO - Executing script: /opt/airflow/scripts/data_preprocessing.py
[2025-04-18T04:04:10.814+0000] {data_pipeline.py:53} INFO - Successfully executed: /opt/airflow/scripts/data_preprocessing.py
[2025-04-18T04:04:10.816+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-18T04:04:10.829+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-18T04:04:10.830+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=clean_data, run_id=scheduled__2025-04-18T03:00:00+00:00, execution_date=20250418T030000, start_date=20250418T040215, end_date=20250418T040410
[2025-04-18T04:04:10.876+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T04:04:10.892+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-18T04:04:10.897+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
